pkg/systemd/logs.jsx

```diff
@@ -55,36 +55,37 @@ import "./logs.scss";
 const _ = cockpit.gettext;
 
 const timeFilterOptions = [
-    { key: "boot", value: 0, toString: () => _("Current boot"), },
-    { key: "boot", value: "-1", toString: () => _("Previous boot") },
-    { key: "since", value: "-24hours", toString: () => _("Last 24 hours"), default: true },
-    { key: "since", value: "-7days", toString: () => _("Last 7 days") },
+    { value: { key: "boot", value: 0 }, content: _("Current boot") },
+    { value: { key: "boot", value: "-1" }, content: _("Previous boot") },
+    { value: { key: "since", value: "-24hours", default: true }, content: _("Last 24 hours") },
+    { value: { key: "since", value: "-7days" }, content: _("Last 7 days") },
```

Comment:
How are these not duplicate React keys?

I know the old code had this as well, and maybe that's just misleading -- these may not actually be React keys, but a custom value that we use to construct the `journalctl` argv? But then, the [`value` is being used as the React key](https://github.com/cockpit-project/cockpit/pull/21356/commits/0ac10078c22d4218ff170f121f36c4fd9f542e86#diff-cde3c8035b0caedbd25d7c632e9be44b71fca3cd18e6905221761695d3a73051R101)  And that should be a string, not an object. How does this work?